HoI4Mod/Rome_Strikes_Back/events/TESB_IND.txt

```diff
@@ -208,4 +222,136 @@ country_event = {
 	option = {
 		name = tesb_india.24.a
 	}
+}
+
+# Burma and Pegu unite
+country_event = {
+	id = tesb_india.25
+	title = tesb_india.25.t
+	desc = tesb_india.25.d
+	picture = GFX_report_event_ger_civil_war
+
+	is_triggered_only = yes
+
+	option = {
+		name = tesb_india.25.a
+		PEG = {
+			transfer_state = 74

```

Comment:
Probably better to use
```		annex_country = {
			target = BRM
			transfer_troops = no/yes (yes for a more 'peaceful' annexation)
		}
```
I also wouldn't remove BRM cores, but just add PEG ones